src/smartpeak/include/SmartPeak/core/SharedProcessors.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <cstdlib> // C2039
```

Comment:
Only the include for `string` should be needed.  I don't believe `cstdlib` is needed.

Reply:
Ok, I will try without on the CI